### DISCUSSION TITLE
`options.fields` removing a property of an instance

### DISCUSSION THREAD

**Discussion by arkapratimc** [sequelize/sequelize#16446] [Help & Questions]
Glitch link - https://glitch.com/edit/#!/sequelize-doubt

After giving the fields option in `Model.create()` method, the `isAdmin` property gets removed, shouldn't it be isAdmin: false ? since the defaultValue is false in the Test Model

Shall the community people kindly explain me why this is happening? I've read this -> https://sequelize.org/docs/v6/core-concepts/model-querying-basics/#simple-insert-queries

![image](https://github.com/sequelize/sequelize/assets/105232141/10d66393-bc33-4c87-a29d-05a91b780309)

**Comment by arkapratimc** [comment]
Any help ?

**Comment by ephys** [comment]
That's what the `fields` option does, it limits which properties can be updated: https://sequelize.org/docs/v7/querying/update/#updating-a-row-using-modelupdate
